system/doc/programming_examples/funs.xmlsrc

```diff
@@ -115,7 +115,7 @@ foreach(fun(Pid) -> Pid ! M end, L)</code>
 
   <section>
     <title>The Syntax of Funs</title>
-    <p>Funs are written with the syntax:</p>
+    <p>Funs are written with the syntax (read more for the full description in <seealso marker="doc/reference_manual:expressions#funs">Fun Expressions</seealso>):</p>
```

Comment:
Maybe "see <seealso.....</seealso> for full description):"? 

Also, please add line breaks to match the rest of the file.